config.json

```diff
@@ -1,7 +1,29 @@
 {
```

Comment:
i'm not sure why this config.json is being checked in - we should just remove this file entirely